freqtrade/commands/cli_options.py

```diff
@@ -463,6 +463,13 @@ def __init__(self, *args, **kwargs):
         choices=['full', 'minimal', 'advanced'],
         default='full',
     ),
+    "leverage": Arg(
```

Comment:
I'm not sure about using an additional argument for this ...

My initial idea here was to use the `--template` flag, and have it included in the advanced template (just as all the other callbacks).
Do you see any problem with that approach? (maybe we can add a `leverage` mode to that if you think this must be separate).

It's just as likely that a user will be using leverage than that they need to write a custom exit (or most other callbacks for that matter) - so for alignment, we would have to have a flag for every callback (which would be quite bad from a user perspective).

An alternative (if we want to make a customized strategy) would be to "ask" the user interactively (similar to `new-config`) - but i'm not sure if the time investment for that is justified - as it's  just as easily get the full template and delete what's not needed.

Reply:
Thanks for your review. Now that I think about it again, maybe this is as simple as, changing signal 'buy' for 'enter_long' in normal templates, and create a new template with only 'enter_short' signals instead. I was unaware about the deprecation of "buy" in favour of "enter_long". But again, you could have both "enter_long" and "enter_short" in a strategy, so how do you solve that using only the "template" argument?

Reply:
is it a problem to have both ways? (maybe the "enter_short" commented out with a comment that this will apply for shorts).

It's anyway intended as a template - not necessarily as something you can use directly - so modifications are kindof expected.